### ISSUE TITLE
Label annotations are wrong in the data upload API

### ISSUE THREAD

**Issue by youichiro** [doccano/doccano#1720]
How to reproduce the behaviour
---------
Annotated label information was incorrect after multiple calls to data upload API via doccano-client.

endpoints:
- post data upload API: `post /v1/projects/{project_id}/upload`
- get spans API: `GET /v1/projects/6/examples/{example_id}/spans`
- get span-types API: `GET /v1/projects/{project_id}/span-types`

get spans API's response example:
```
[
  {
    created_at: "2022-02-25T09:01:34.992298Z"
    end_offset: 5
    example: 2263
    id: 2371
    label: 2470
    prob: 0
    start_offset: 3
    updated_at: "2022-02-25T09:01:34.992336Z"
    user: 1
  },
]
```

problem:
- `label: 2470` but span-types list does not contain the id.

Your Environment
---------
doccano v1.5.5

**Comment by Hironsan** [comment]
I want to try to reproduce the problem. Can you show me some example data?

**Comment by youichiro** [comment]
Keep doccano running.
In doccano-client repository, prepare sample data file and run the following python script.

`./data/demo.jsonl`
```./data/demo.jsonl
{"text": "今日は小川さんの出番だ", "labels": [ [3, 5, "人名"] ]}
{"text": "小川さんの電話番号は999-9999-9999だ", "labels": [ [0, 2, "人名"], [10, 23, "電話番号"] ]}
{"text": "高校 体育（合同） 1-1、1-2", "labels": [ [10, 13, "住所"], [14, 17, "住所"] ]}
```

`./demo_upload.py`
```python
from doccano_api_client import DoccanoClient

doccano_client = DoccanoClient(
  'http://localhost',
  'admin',
  'password'
)

def create_project_and_upload(index: int):
  # create a new project
  new_project = doccano_client.create_project(
    name=f'project_{index}',
    description='description',
    project_type='SequenceLabeling',
    guideline='guideline',
    resourcetype='SequenceLabelingProject',
    randomize_document_order=False,
    collaborative_annotation=True,
  )
  project_id = new_project['id']

  # upload data
  jsonl_upload = doccano_client.post_doc_upload(
    project_id=project_id,
    file_name='demo.jsonl',
    file_path='./data/',
    column_data="text",
    column_label="labels",
    encoding="utf_8",
    format="JSONL",
  )

  # create label
  doccano_client.create_span_type(
    project_id=project_id,
    text='人名',
  )

for i in range(3):
  create_project_and_upload(i+1)
```

```sh
python demo_upload.py
```

Opening annotation page, wrong labels are displayed.

![スクリーンショット 2022-03-08 9 16 53](https://user-images.githubusercontent.com/20487308/157140715-224cef22-6692-4656-8ebe-14f08857b2cb.png)

**Comment by Hironsan** [comment]
Thanks! But I can't reproduce the problem in v1.5.5.

![image](https://user-images.githubusercontent.com/6737785/157167321-3ee369a5-7978-4c37-b33b-21e58599f436.png)

In addition, doccano v1.5.5 does not have the `span-types` and `spans` endpoints. Because they are introduced in https://github.com/doccano/doccano/pull/1619 on 2022/1/7(v1.5.5 was released 2021/12/3).

And I tried to increase the number of the data(because you are using 132 data), but there is no problem. Maybe another reason.
